css/style.css

```diff
@@ -16,4 +15,107 @@ body {
  * - When using Flexbox, remember the items you want to move around need to be inside a parent container set to 'display: flex'
  */
 
+ .header,
+ .ma-img,
+ .footer,
+ .icon{
+    padding: 0 1.5rem;
+    max-width: auto;
+    margin: 0 auto;
+ }
+ .header{
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    height: 5rem;
+    margin-bottom: 1rem;
+    padding-top: 1rem;
+ }
+ nav{
+    display: flex;
+    width: 100%;
+ }
+.navbar{
+    display: flex;
+    list-style: none;
+    font-size: normal;
+    color: lightgrey;
+    padding-left: 8rem;
+   
+}
+.nav-li-a{
+    margin: 20px;
+    text-decoration: none;
+    color: gray;
+}
+
+.karma-logo{
+    max-height: 40px;
+    width: auto;
+    padding-right: 10rem;
+}
+.meet{
+    font-weight: bold;
+}
+.Store{
+    color: orange;
+}
 
+.ma-img{
+    display: flex;
+    background-image: url( "/img/first-background.jpg");
+    height: 500px;
+    background-size: cover;
+    color: white;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+}
+.learn-more{
+    background-color: rgb(235, 114, 66);
+    border-radius: 10px;
+    padding: 10px 20px;
+    margin-bottom: 5rem;
```

Comment:
This Button is a bit off I'm afraid. Has a white border and the border-radius is too big so it's more rounded than it outght to be